Meetings/2018-04-18.md

```diff
@@ -1,125 +1,126 @@
+April 18, 2018
+9am-10:30am EST
+
+## To join the online meeting
 
 1. Go to https://pwgsc-nh.webex.com/pwgsc-nh/j.php?MTID=m434545e71be581d1e13986f819b47a08
-2. Enter your name and email address. 
+2. Enter your name and email address.
 3. Enter the meeting password: 9Taub6D8
-4. Click "Join Now". 
+4. Click "Join Now".
+
 Please feel free to use the Webex chat box to relay any questions.
 
-## To join the teleconference only 
+## To join the teleconference only
 
-Provide your phone number when you join the meeting to receive a call back. Alternatively, you can call: 
+Provide your phone number when you join the meeting to receive a call back. Alternatively, you can call:
 
-Call-in toll-free number: 1-877-413-4781  (Canada) 
+Call-in toll-free number: 1-877-413-4781  (Canada)
 
-Call-in number: 1-613-960-7510  (Canada) 
+Call-in number: 1-613-960-7510  (Canada)
 
-Attendee access code: 320 160 2 
+Attendee access code: 320 160 2
 
-## For assistance 
- 
-1. Go to https://pwgsc-nh.webex.com 
-2. On the left navigation bar, click "Support". 
-3. Call 1-800-226-6338 or 613-941-9554 
+## For assistance
 
-If you have any inquiries or concerns, you can contact: 
-ashley.casovan@tbs-sct.gc.ca 
+1. Go to https://pwgsc-nh.webex.com
+2. On the left navigation bar, click "Support".
+3. Call 1-800-226-6338 or 613-941-9554
 
-Add this meeting to your calendar (Cannot add from mobile devices): 
+If you have any inquiries or concerns, you can contact:
+ashley.casovan@tbs-sct.gc.ca
+
+Add this meeting to your calendar (Cannot add from mobile devices):
 https://pwgsc-nh.webex.com/pwgsc-nh/j.php?MTID=m434545e71be581d1e13986f819b47a08
 
-## Agenda 
-1. Announce Working Groups:
+## Agenda
 
-   a. Rules
-   
-   b. Tools
-   
-   c. People
-2. Open Source Day
-3. Agile Procurement Presentation 
-4. Github Best Practices Presentation 
-5. Roundtable 
+1. Announce Working Groups:
+   1. Rules
+   1. Tools
+   1. People
+1. Open Source Day
+1. Agile Procurement Presentation
+1. Github Best Practices Presentation
+1. Roundtable
 
-## 1.Announce Working groups: 
+## 1.Announce Working groups
 
 * To view the presentation please visit the [Open Source Working Groups April 18 Presentation Slidewiki website](http://slidewiki.aksw.org/deck/30-1)
 
-   a. Rules
-   
-   b. Tools
-     * Jack will lead.
-     * Broad topic: Provide more understanding of peoples technologies. Looking for collaboration: business and technical representation in this group. Work in the working groups and present their work the OSAB.
-     * Propose: move the meeting around. First one in Kanata and then alternate different locations.   
-     
-   c. People
-   
-For the working groups, there will be support from CIOB and the GCTools team. The Open Source Secretriat will be present at all of the working groups meeting. The working groups will give a presentation of the work that they have done at the beginning of each Open Source Advisory Board meeting. 
-   
+1. Rules
```

Comment:
I think changing the list style here is the only "functional" change. using the `1.` changes it to an ordered list with numbers at the first level, and `i`'s for the secont